pyproject.toml

```diff
@@ -68,6 +68,7 @@ requests-aws4auth = "^1.2.3"
 psycopg2-binary = "^2.9.9"
 urllib3 = "<2.0.0"
 SQLAlchemy = "<=3.0.0"
+imap-data-access = ">=0.3.1"
 
 # Used for installing with pip
 [tool.poetry.extras]
```

Comment:
Is this intentionally deleted?

Reply:
It's not deleted. It's highlighted in weird way.

Reply:
Oh, odd 🤷